core/shared/src/main/scala/zio/stm/ZSTM.scala

```diff
@@ -1621,15 +1621,17 @@ object ZSTM {
      * Creates a function that can reset the journal.
      */
     def prepareResetJournal(journal: Journal): () => Any = {
```

Comment:
The extra set of braces can now be removed.

Reply:
Sure, Thanks. I will do that.

Reply:
I just added a new commit removing the extra set of braces. Thanks!